### DISCUSSION TITLE
Create PR only for certain branches immediately

### DISCUSSION THREAD

**Discussion by 4soft-liebig** [renovatebot/renovate#26674] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.115.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

Hi,
we are using renovate bot with several base-branches `["develop", "/^hotfix\\/.*/"]` to update dependencies for different managers (maven, npm, ...).

In our setup PRs are created immediately, which is sufficient for `develop`. However, for any `hotfix`-branch, we would like to create PRs manually. The major reason is, that we don't want the "noise" of getting many PRs created by renovate bot for an older version, that we still have to support (i.e., for which we have to create hotfixes). We would rather "opt-in" for dependency updates on any `hotfix`-branch via the dependency dashboard.

Trying different configuration options (`dependencyDashboardApproval` and `prCreation`) as part of renovate-bot's `packageRule` section seem not to be supported (in combination with `matchBaseBranches`), and are most likely the wrong abstraction ("packages" vs. "branches"). Is there a way to configure the setup stated above?

One additional information: In our setup we run renovate-bot triggered PRs the same way as any regular feature- or fix-branch as a standard GitLab merge-request pipeline.

Best regards
Jörg

**Comment by viceice** [comment]
`dependencyDashboardApproval ` should work in a package rule with `matchBaseBranches`

**Reply by 4soft-liebig** [reply]
Thanks for the fast response.

Activating `dependencyDashboardApproval`: What should it look like? Do corresponding dependencies appear in the 'awaiting-schedule' section and I have to check corresponding dependencies, so that PRs get created on the next renovate-bot run?

![image](https://github.com/renovatebot/renovate/assets/102736080/23e02564-8386-44b8-b1b8-dc26002d8f42)

**Reply by viceice** [reply]
no, you should have a pending approval section

![image](https://github.com/renovatebot/renovate/assets/1798109/ad06dcb6-6fa0-4379-8201-81153be1ebac)


but only if the branch / pr doesn't exit yet.

**Reply by 4soft-liebig** [reply, marked as answer]
Thank you @viceice,

the following configuration snippet works as expected

```
  "packageRules": [
    {
      "matchBaseBranches": ["/^hotfix\\/.*/"],
      "dependencyDashboardApproval": true
    },
  ],
```

... and creates the corresponding section in the dependency dashboard.

![image](https://github.com/renovatebot/renovate/assets/102736080/68056902-32d1-42a5-a5cb-5016d194f592)
